airbyte-cdk/python/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.1.10
+Add checking specified config againt spec for read, write, check and ddiscover commands
+
+##0.1.9
+remove this line after rebase
```

Comment:
rudiment?

Reply:
No, its cause Artur has his MR with 0.1.9 version published so my is  0.1.10 https://pypi.org/project/airbyte-cdk/

Reply:
I'm waiting when he will be ready, rebase my changes onto his and update this file